src/query/formats/src/file_format_type.rs

```diff
@@ -335,6 +345,9 @@ fn final_csv_options(options: &mut FileFormatOptionsExt) -> Result<()> {
     if options.quote.is_empty() {
         options.quote = "\"".to_string();
     }
+    if options.stage.nan_display.is_empty() {
```

Comment:
other formats need to deal with it too. because they will use `nan_display` later
and `nan_display=""` will be store in meta and cause problem.

Reply:
other formats using consts right now, will modify file format one by one.